tests/unit/pie-test.js

```diff
@@ -168,20 +249,129 @@ test('Percentage labels can have decimal points', function(assert) {
   //
   // NOTE: There could be fewer decimal places than maxDecimalPlace if the percentage
   // happens to be a whole number
-  var verifyDecimalPlaces = function(items) {
+  var verifyDecimalPlaces = function(items, maxDecimalPlace) {
     for (var i = 0; i < items.length; i++) {
-      assert.ok(countDecimals(items[i].percent) <= pieContent.maxDecimalPlace);
+      assert.ok(countDecimals(items[i].percent) <= maxDecimalPlace);
     }
   };
-  //-------------------- End Setup ------------------------//
 
-  pieContent.maxDecimalPlace = 2;
-  var component = this.subject(pieContent);
+  var component = this.subject(pieWithDecimal);
 
   var finishedData = this.subject().get('finishedData');
-  var otherItems = finishedData[0]._otherItems;
-  var pieItems = finishedData.slice(1, finishedData.length);
+  var pieItems = [];
+  var otherItems = [];
+
+  for (var i = 0; i < finishedData.length; i++) {
+    if (finishedData[i].label === "Other") {
+      otherItems.push.apply(finishedData[i]._otherItems);
+    } else {
+      pieItems.push(finishedData[i]);
+    }
+  }
+
+  verifyDecimalPlaces(pieItems, pieWithDecimal.maxDecimalPlace);
+  verifyDecimalPlaces(otherItems, pieWithDecimal.maxDecimalPlace);
+});
+
+test('There are no label overlap in the pie charts', function(assert) {
+  //-------------------- Begin Setup ------------------------//
+  var pieThinSliceContent = {
+    maxRadius: 200,
+    maxNumberOfSlices: 10,
+    minSlicePercent: 0,
+    data: [
+      {
+        "label": "Label 1",
+        "value": 20
+      },
+      {
+        "label": "Label 2",
+        "value": 50
+      },
+      {
+        "label": "Label 3",
+        "value": 10
+      },
+      {
+        "label": "Label 4",
+        "value": 40
+      },
+      {
+        "label": "Label 5",
+        "value": 0.1
+      },
+      {
+        "label": "Label 6",
+        "value": 0.2
+      },
+      {
+        "label": "Label 7",
+        "value": 0.3
+      },
+      {
+        "label": "Label 8",
+        "value": 18
+      },
+      {
+        "label": "Label 9",
+        "value": 18
+      },
+      {
+        "label": "Label 10",
+        "value": 25
+      }
+    ]
+  };
+
+  // Gets the transformed coordinates of the pie chart labels
+  // (0,0) is the center of the pie
+  var getCoordinates = function(label) {
+    var xforms = label.transform.baseVal;
+    var firstXForm = xforms.getItem(0);
+    var firstX = firstXForm.matrix.e;
+    var firstY = firstXForm.matrix.f;
+    return [firstX, firstY];
+  };
+
+  // Given a list of Y-Coordinates for one side of a pie, detects whether there
+  // are any label intersections.
+  var assertNoLabelOverlap = function(yCoordinateList, labelHeight) {
+    yCoordinateList = yCoordinateList.sort(function sorter(a, b) {
```

Comment:
Might be cleaner to make `sorter` an anonymous function, as it doesn't get used outside the scope of `sort`